### PR TITLE
Add docs for fetching data from a URL

### PR DESCRIPTION
**Pull request by jtpio**
## References

Fixes https://github.com/jupyterlite/jupyterlite/issues/919

## Code changes

Update the documentation to mention the use of the `pyodide-http` package which patches `requests` and `urllib`.

## User-facing changes

Documentation changes.

## Backwards-incompatible changes

None

### CODE REVIEW

**Comment by jtpio** [comment]
cc @startakovsky if you want to have a look: https://jupyterlite--930.org.readthedocs.build/en/930/howto/content/python.html#fetching-remote-content

File: docs/howto/content/python.md [hunk 1]
```diff
@@ -39,3 +39,41 @@ os.getcwd()  # If successful:  "/drive/path/to/notebook"
...
+filename = 'data.csv'
+
+with open(filename, 'w') as f:
+    f.write(text)
+
+data = pd.read_csv(filename, sep=',')
+data
+```
+
+As an alternative, you can also install the `pyodide-http` package to use familiar
```
Comment (bollwyvl) [inline docs/howto/content/python.md:66]: Perhaps:
- a link to the project page
- note that it's _not_ a first-party package supported by pyodide

File: docs/howto/content/python.md [hunk 2]
```diff
@@ -39,3 +39,41 @@ os.getcwd()  # If successful:  "/drive/path/to/notebook"
   https://github.com/jupyterlite/jupyterlite/blob/main/examples/pyolite/emscripten-filesystem.ipynb
 [fs]: https://emscripten.org/docs/api_reference/Filesystem-API.html
 [caniuse-sw]: https://caniuse.com/serviceworkers
+
+## Fetching remote content
+
+It is also possible to fetch content from remote URL. For example if you are using the
+Pyodide kernel, you can use the `fetch` function to do so:
+
+```py
```
Comment (bollwyvl) [inline docs/howto/content/python.md:48]: Probably add headers here, and have a link to `pyodide.http`, as `py_fetch` is also quite usable:

https://pyodide.org/en/stable/usage/api/python-api/http.html

File: docs/howto/content/python.md (hunk 1 above)
Comment (jobovy) [reply docs/howto/content/python.md:66]: It was actually just added: https://github.com/pyodide/pyodide/pull/3355.

**Pull request closed without merging**